lib/cardano-sl.cabal

```diff
@@ -97,20 +92,18 @@ library
                         Pos.Web.Server
                         Pos.Web.Types
 
+                        Pos.Delegation.HelpersLib
```

Comment:
Why is it here and not near `Pos.Delegation.Listeners` and `Pos.Delegation.Worker`?